_publications/joss/paper.md

```diff
@@ -0,0 +1,162 @@
+bases or do not support or distribute the code bases to the wider
+community. PFHub is a community effort spearheaded by the Center for
+Hierarchical Materials Design at Northwestern University and the
+National Institute of Standards and Technology to support the
+development of phase-field codes. The goal of PFHub is to improve
+cross-collaboration between phase-field code developers and
+practitioners by providing a standardized set of benchmark problems
+[@bm1, @bm2] along with a web framework for uploading and comparing
+benchmark results from different codes.
+
+## Website
+
+The PFHub website provides a facility for uploading, displaying and
+comparing results from the benchmark problems. The website uses the
+Jekyll static website generator along with automated frontend
+processing to eliminate the need for content management systems
+[@cmsfree], which are generally costly to maintain especially for
+small scientific communities with limited funding and manpower.
+
+The workflow for uploading benchmark results relies on third party
+tools using the following steps, illustrated in Fig. 1.
+
+- The users are first required to archive simulation outputs at an
+  archival resource (*e.g.*, Figshare) configured with permissive
+  cross-origin resource sharing (CORS).
+- The metadata summarizing each simulation is entered into a form on
+  the website, including relevant details such as memory usage, run
+  time and links to the data archived in the first step.
+- Upon submission, the Staticman app [@staticman] submits the entered
+  metadata as a GitHub pull request to the PFHub GitHub repository.
+  The metadata is stored in a YAML file with a unique path in the
+  repository.
+- Travis CI [@travis] performs linting on the submission and then
+  launches a new version of the website using Surge [@surge.sh]. The
+  PFHub admins can then examine the new submission and further changes
+  can be made if necessary.
+- Once review has been completed to the satisfaction of both the 
+  uploading scientist and the website maintainers, the pull request
+  is merged and served to the World Wide Web using a hosting service
+  compatible with GitHub Pages.
+
+A combination of Jekyll templates and Coffeescript is used to access
+and download the data links in the submitted YAML files and then
+display the data in interactive figures on the website. The
+combination of a central repository on GitHub for website source code
+and metadata with distributed data records on third-party archives
+avoids the complexity and administrative overhead of maintaining a
+live database and associated backend app. The PFHub infrastructure
+provides a template for other small scientific communities to host
+custom content and integrate data from members of their community.
+
+![PFHub schematic](./pfhub_website.svg)
+**Figure 1:** Schematic overview of the PFHub framework for building
+scientific research portals, simply.
+
+## Acknowledgments
+
+Acknowledge MGI funding???
```

Comment:
Now's probably a good time to resolve this question.

Reply:
We're not submitting the JOSS paper only the JORS paper.